package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/github": "^4.0.0",
+    "lodash": "^4.17.20",
```

Comment:
Hmm, I tend to avoid using `lodash` as it comes with loads of unnecessary features, many of which are already part of built-in JavaScript functions. But in your case, it may be a good idea to use some 3rd party tool as you need to implement something equivalent to `sampleSize`.

Let me give it a think (I'm still on holiday, so I'm a bit slow to respond this week though).

Reply:
Since there seems to be no good way to randomly sample array elements by default in JS, using Lodash seems to be the easiest and simplest way.